src/main/scala/zio/actors/ActorSystem.scala

```diff
@@ -269,6 +279,17 @@ private[actors] object ActorSystemUtils {
         )
     }
 
+  def buildFinalName(parentActorName: String, actorName: String): Task[String] =
+    actorName match {
+      case ""   => IO.fail(new Exception("Actor actor must not be empty"))
+      case null => IO.fail(new Exception("Actor actor must not be null"))
+      case name =>
+        regexName.pattern.matcher(name).matches match {
```

Comment:
You can use regex directly:

```scala
actorName match {
  case null => ...
  case "" => ...
  case regexName(_*) => ..
  case _ => ...
}
```

Reply:
Great! I didn't know I could use it like that. Thanks